toolboxes/feel/feelmodels/modelcore/terminalsize.hpp

```diff
@@ -0,0 +1,12 @@
+/* -*- mode: c++; coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; show-trailing-whitespace: t -*- vim:fenc=utf-8:ft=cpp:et:sw=4:ts=4:sts=4
+ */
+
+#pragma once
+
+#include <utility>
+
+namespace Feel
+{
+std::pair<int,int> get_terminal_size();
```

Comment:
why not in feelcore ? it might be useful from there with tabulate

Reply:
yes, it's a first draft. It's planned (I use only the Feel namespace)